lib/libsm/include/sm2.h

```diff
@@ -0,0 +1,46 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2019 Huawei Technologies Co., Ltd
+ */
+/*
+ *  sm2_signature.h
+ *  mbedtlsSM2
+ *
+ *  Created by mac on 2018/4/18.
+ *  Copyright © 2018年 mac. All rights reserved.
```

Comment:
I kept the copyright notice unmodified on purpose, because it was written like this by the author and I did not feel the right to change it, it's a bit like a signature if you will.

That being said, we would not lose anything by dropping the `©` and the `年` (nián = "year" in chinese).